cmd/tetra/getevents/getevents.go

```diff
@@ -227,5 +249,7 @@ redirection of events to the stdin. Examples:
 	flags.BoolVar(&Options.ImaHash, "ima-hash", true, "Include ima hashes in compact output")
 	flags.StringSliceVar(&Options.PolicyNames, "policy-names", nil, "Get events by tracing policy names")
 	flags.StringSliceVar(&Options.CelExpression, "cel-expression", nil, "Get events satisfying the CEL expression")
+	flags.BoolVar(&Options.Reconnect, "reconnect", false, "Keep trying to connect even if an error occurred")
+	flags.DurationVar(&Options.ReconnectTimeout, "reconnect-timeout", 2*time.Second, "reconnect timeout")
```

Comment:
Just a semantic question: isn't a timeout a time until the thing end completely? Shouldn't it be named a backoff or something like that?

Reply:
How about `reconnect-wait`?

Reply:
sure

Reply:
sorry for spotting this just now after the first review